bin/ruby-build

```diff
@@ -701,7 +701,7 @@ fix_rbx_irb() {
 }
 
 require_java7() {
-  local version="$(java -version 2>&1 | grep 'java version' | head -1)"
+  local version="$(java -version 2>&1 | grep '\(java\|openjdk\) version' | head -1)"
```

Comment:
@chnoack Thanks for reporting. It seems that `grep` on your system doesn't support the regular expression used here?